pkg/projects/dir.props

```diff
@@ -49,6 +49,11 @@
     </Otherwise>
   </Choose>
 
+  <ItemGroup>
+    <AdditionalLibPackageExcludes Condition="'$(SymbolFileExtension)' != ''" Include="%2A%2A\%2A$(SymbolFileExtension)" />
```

Comment:
That is totally not obvious to someone just looking at this code.

In all honesty, it would probably be best to not use `Include/ItemSpec` in MSBuild in these situations.  There are just too many bugs that pop up because MSBuild has certain semantics on `Include`.